client/src/components/territoryFocusControl.tsx

```diff
@@ -1,54 +1,18 @@
 import {useIntl} from 'gatsby-plugin-intl';
 import React from 'react';
-import {LngLatBoundsLike} from 'mapbox-gl';
+import {MapEvent} from 'react-map-gl';
 
 import * as styles from './territoryFocusControl.module.scss';
 import * as EXPLORE_COPY from '../data/copy/explore';
-import * as constants from '../data/constants';
 
 interface ITerritoryFocusControl {
-  goToPlace(bounds: LngLatBoundsLike): void;
+  onClick(event: MapEvent | React.MouseEvent<HTMLButtonElement>):void;
```

Comment:
Slick 👍

Reply:
Thx! This is still kinda head scratcher as to why the IDE complains about the `feature` property not being in the event payload. I have to look into this. Not a show stopper but more of a personal preference to not have any linting warnings.